README.md

```diff
@@ -6,13 +6,9 @@ before spawning Ruby processes.
 
 ## Installation
 
-To install rbenv-vars, clone this repository into your
-`~/.rbenv/plugins` directory. (You'll need a recent version of rbenv
-that supports plugin bundles.)
+Make sure you have the latest rbenv and ruby-build versions, then run:
```

Comment:
Good catch... the copy-n-paste :imp: strikes again!  Removed in 385750f9943aea304e40dd97a38456c70655619a